tests/contrib/operators/test_ecs_operator.py

```diff
@@ -95,7 +95,7 @@ def test_init(self):
         self.aws_hook_mock.assert_called_once_with(aws_conn_id=None)
 
     def test_template_fields_overrides(self):
-        self.assertEqual(self.ecs.template_fields, ('overrides',))
```

Comment:
This is a senseless test. Tests should check behavior, not field values.
It's better to write something like: https://github.com/apache/airflow/blob/master/tests/contrib/operators/test_gcp_compute_operator.py#L73-L98

Reply:
@mik-laj Not different than the test in this PR https://github.com/apache/airflow/pull/5490 which was merged. So why hold this PR stale?